app/controllers/foreman_tasks/tasks_controller.rb

```diff
@@ -122,9 +122,11 @@ def resource_scope(_options = {})
       @resource_scope ||= ForemanTasks::Task.authorized("#{action_permission}_foreman_tasks")
     end
 
+    # rubocop:disable Naming/MemoizedInstanceVariableName
     def find_dynflow_task
-      resource_scope.where(:type => 'ForemanTasks::Task::DynflowTask').find(params[:id])
+      @dynflow_task ||= resource_scope.where(:type => 'ForemanTasks::Task::DynflowTask').find(params[:id])
```

Comment:
I believe you do not need the memoization here (you're always calling it just once, right?) and it would prevent rubocop from complaining.